server-tests/tests/ws_test.rs

```diff
@@ -51,7 +51,7 @@ async fn can_request_underlying_websocket_server(
     let msg = "Hello, WebSocket!";
     ws_stream
         .send(tokio_tungstenite::tungstenite::Message::Text(
-            msg.to_string(),
+            msg.to_string().into(),
         ))
```

Comment:
Tungstenite `Utf8Bytes` [implement from &str](https://docs.rs/tungstenite/latest/tungstenite/protocol/frame/struct.Utf8Bytes.html#impl-From%3C%26str%3E-for-Utf8Bytes), so this should be enough:

```rust
    ws_stream
        .send(tokio_tungstenite::tungstenite::Message::Text(msg.into()))
        .await
        .expect("Failed to send message");
```